src/driver/ixgbe.c

```diff
@@ -51,6 +56,213 @@ struct ixgbe_tx_queue {
 	void* virtual_addresses[];
 };
 
+/**
+ * ixgbe_set_ivar - set the IVAR registers, mapping interrupt causes to vectors
+ * @param dev pointer to device
+ * @param direction 0 for Rx, 1 for Tx
+ * @param queue queue to map the corresponding interrupt to
+ * @param msix_vector the vector to map to the corresponding queue
+ */
+static void set_ivar(struct ixgbe_device *dev, s8 direction, u8 queue, u8 msix_vector) {
```

Comment:
please use C++/Stroustrup style for pointers: `foo* bar`, not `foo *bar` to be consistent with existing code